### DISCUSSION TITLE
How WebIO registers dependencies

### DISCUSSION THREAD

**Discussion by neciszhang** [donmccurdy/glTF-Transform#811] [Q&A] [answered]
Hi, Don,
        I use webIO in browser, i want register draco or meshopt dependencies,how do I register dependencies.
Thanks

**Comment by donmccurdy** [comment, marked as answer]
Hi @neciszhang — the [DracoMeshCompression](https://gltf-transform.donmccurdy.com/classes/extensions.dracomeshcompression.html) and [MeshoptCompression](https://gltf-transform.donmccurdy.com/classes/extensions.meshoptcompression.html) documentation should explain how to register the dependencies, you can replace NodeIO with WebIO. The other obstacle is that the Draco npm package does not work on the web. I think you'll have to copy the decoders into your web application, as described in this response:

https://stackoverflow.com/questions/66975407/convert-glb-to-draco-glb-front-end/66978236#66978236
